benches/raw_insertions.rs

```diff
@@ -2,10 +2,10 @@ use criterion::{criterion_group, criterion_main, Criterion};
 use rand::Rng;
 use std::hint::black_box;
 use vex_prover::imt::order::Order;
-use vex_prover::imt::IndexedMerkleTree;
+use vex_prover::imt::SellIMT;
 
 fn insert_batch(n: u64) {
-    let mut imt = IndexedMerkleTree::new();
+    let mut imt = SellIMT::new();
```

Comment:
The benchmark is now hard-coded to use only SellIMT. Since the PR's goal is to support both buy and sell sides with a generic IMT, consider modifying the benchmark to test both sides or to use the generic version with a type parameter.

If you apply the previous suggestion to import IndexedMerkleTree, you could use:
```suggestion
    let mut imt = IndexedMerkleTree::<_, Sell>::new();
```

Reply:
fixed this in [pr/12](https://github.com/aerius-labs/vex-prover/pull/12)